extensions/amp-list/0.1/amp-list.js

```diff
@@ -375,6 +397,31 @@ export class AmpList extends AMP.BaseElement {
   }
 }
 
+/** @private */
+function onFetchSuccess_() {
+  if (this.getFallback()) {
+    // Hide in case fallback was displayed for a previous fetch.
+    this.toggleFallbackInMutate_(false);
+  }
+  this.togglePlaceholder(false);
+  this.toggleLoading(false);
+}
+
+/**
+ * @param {!Error} error
+ * @private
+ * @throws {!Error} throws error if fallback element is not present.
+ */
+function onFetchError_(error) {
+  this.toggleLoading(false);
+  if (this.getFallback()) {
+    this.toggleFallbackInMutate_(true);
+    this.togglePlaceholder(false);
+  } else {
+    throw error;
```

Comment:
@jridgewell @choumx Should this not be a `user().error`? When is it appropriate to `throw` vs `user()` errors

Reply:
yea good point. I'll wait for the their response but this should be a user().error as I don't see a catch statement handling this. Note that this was the existing behavior..as I just extracted to a method for reuse.